znnl/training_strategies/simple_training.py

```diff
@@ -369,7 +369,7 @@ def train_model(
         state = self.model.model_state
 
         loading_bar = trange(
-            1, epochs + 1, ncols=100, unit="batch", disable=self.disable_loading_bar
+            0, epochs, ncols=100, unit="batch", disable=self.disable_loading_bar
```

Comment:
Is this relevant?

Reply:
If you want a recorder to only take the model's state at initialization, then yes, it is. I think this is the more flexible form and is, therefore, better.